libs/core/src/lib/combobox/combobox.component.html

```diff
@@ -6,38 +6,58 @@
             [closeOnOutsideClick]="closeOnOutsideClick"
             class="fd-combobox-popover-custom">
     <fd-popover-control>
-        <div class="fd-combobox-control">
-            <fd-input-group [glyph]="glyph"  [compact]="compact" [button]="true" [state]="state" [inShellbar]="inShellbar"
-                            (addOnButtonClicked)="onPrimaryButtonClick()" [buttonFocusable]="buttonFocusable">
-                <input #searchInputElement type="text"
-                       fd-input-group-input
-                       [compact]="compact"
-                       (keydown)="onInputKeydownHandler($event)"
-                       (keyup)="onInputKeyupHandler($event)"
-                       [disabled]="disabled"
-                       [(ngModel)]="inputText"
-                       (ngModelChange)="handleSearchTermChange()"
-                       placeholder="{{placeholder}}">
-            </fd-input-group>
-        </div>
+        <fd-input-group [glyph]="glyph"  [compact]="compact" [button]="true" [state]="state" [inShellbar]="inShellbar"
+                        (addOnButtonClicked)="onPrimaryButtonClick()" [buttonFocusable]="buttonFocusable" [disabled]="disabled">
+            <input #searchInputElement type="text"
+                   fd-input-group-input
+                   [compact]="compact"
+                   (keydown)="onInputKeydownHandler($event)"
+                   (keyup)="onInputKeyupHandler($event)"
+                   [disabled]="disabled"
+                   [(ngModel)]="inputText"
+                   (ngModelChange)="handleSearchTermChange()"
+                   [placeholder]="placeholder">
+        </fd-input-group>
     </fd-popover-control>
     <fd-popover-body *ngIf="displayedValues && displayedValues.length">
-        <fd-menu class="fd-combobox-input-menu-overflow" [style.maxHeight]="maxHeight">
+        <ul fd-list [dropdownMode]="true" class="fd-combobox-input-menu-overflow" [style.maxHeight]="maxHeight" [hasMessage]="listMessages.length > 0">
             <ng-content></ng-content>
-            <ul fd-menu-list>
+            <ng-container *ngIf="groupFn">
+                <ng-container *ngFor="let group of displayedValues | listGroupPipe:groupFn | keyvalue; let groupIndex = index">
```

Comment:
It looks like expected type of `group` is: `{key: any, value: any[]}`
But I dont see proper typings for:
```
    @Input()
    groupFn: Function;
```
```
    /** @hidden */
    displayedValues: any[] = [];
```
```
    listGroupPipe
```